api/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:13-alpine
+
+COPY nest-cli.json package.json tsconfig.json tsconfig.build.json tslint.json ./
+
+WORKDIR /app
+
+COPY . /app/
```

Comment:
Looking at the package.json file, will need a `node_modules` and `src` directory